lib/gems/pending/appliance_console/database_replication.rb

```diff
@@ -52,6 +53,10 @@ def create_config_file(host)
         f.puts("conninfo='host=#{host} user=#{database_user} dbname=#{database_name}'")
         f.puts("use_replication_slots=1")
         f.puts("pg_basebackup_options='--xlog-method=stream'")
+        f.puts("failover=automatic\n")
```

Comment:
👍  I like this idea. Wish I would have thought of it. ;) I could have used this approach on some of my appliance_console PRs.